tests/Transport/TransportBenchmarkTest.php

```diff
@@ -18,8 +18,24 @@
  */
 class TransportBenchmarkTest extends BaseTest
 {
+    /**
+     * @var int
+     */
     protected $_max = 1000;
+
+    /**
+     * @var int
+     */
     protected $_maxData = 20;
+
+    /**
+     * @var array<string, array<string, array{
+     *    count: int,
+     *    max: int,
+     *    min: int,
+     *    mean: float,
+     * }>>
```

Comment:
```suggestion
     * }
```

Reply:
this suggestion is wrong. We need to close the generic types for the 2 levels of arrays after closing the array shape for the inner array